docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -40,6 +40,10 @@ The `./config/database.js` (or `./config/database.ts` for TypeScript) accepts 2
 | `pool`<br /><br />_Optional_                             | [Database pooling options](#database-pooling-options)                                                 | `Object`  | -       |
 | `acquireConnectionTimeout`<br /><br />_Optional_         | How long knex will wait before throwing a timeout error when acquiring a connection (in milliseconds) | `Integer` | `60000` |
 
+::: note
+The `client` value may be replaced by Strapi before being passed to Knex, based on which database driver packages are available in the project.
```

Comment:
Strapi treats `postgres`, `sqlite`, and `mysql` as generic names for the client you are using, and may replace the `client` value with the best available package before passing it on to Knex if a preferred package is available.

For example, if `better-sqlite3` is installed in your project, Strapi will replace the client value `sqlite` with `better-sqlite3` before creating the Knex connection.